src/axes/numericAxis.ts

```diff
@@ -262,6 +278,26 @@ export module Axes {
       }
     }
 
+    /**
+     * Gets the approximate text width setting.
+     *
+     * @returns {boolean} The current approximate text width setting.
+     */
+    public approximateTextWidth(): boolean;
+    /**
+     * Sets the approximate text width setting.
+     *
+     * @param {boolean} enable faster but less accurate computation of text widths.
+     */
+    public approximateTextWidth(enable: boolean);
+    public approximateTextWidth(enable?: boolean): boolean {
+      if (enable == null) {
+        return this._approximateTextWidth;
+      } else {
+        this._approximateTextWidth = enable;
+      }
```

Comment:
Under the setter case, if we enable the usage of approximation, do we want to rerender the axis again using the approximation?